### PR TITLE
feature/linkCreationSupport 

### LINKED ISSUE #13: Support internal links in SilverStripe
The markdown editor needs to be able to handle references to internal SilverStripe pages.

They should be rendered out as full `Page::Link()` anchors.

### PR DESCRIPTION
**Pull request by matiasnombarasco**
Now you can use a fancy overlay popup to create links and search in the list of already created articles for it

Resolves #13

### CODE REVIEW

File: forms/MarkdownTextareaField.php [hunk 1]
```diff
@@ -34,22 +34,31 @@ class MarkdownTextareaField extends TextareaField
      * @param  array $properties
      * @return string            HTML to be used
      */
-    public function FieldHolder($properties = array())
+    public function FieldHolder($properties = [])
```
Comment (robbieaverill) [inline forms/MarkdownTextareaField.php]: Hey @matiasnombarasco - this module supports PHP 5.3, so best to leave it the way it was

File: thirdparty/codemirror.css [hunk 2]
```diff
@@ -0,0 +1,347 @@
+/* BASICS */
```
Comment (robbieaverill) [inline thirdparty/codemirror.css]: Is there a minified equivalent?

Comment (matiasnombarasco) [inline thirdparty/codemirror.css]: The vendor doesn't provide one, but I can easily compress it, just hold on line please...

**Comment by matiasnombarasco** [comment]
I just amended those change requests @robbieaverill

**Pull request merged**